### ISSUE TITLE
waitUntil not working for Remix.run apps ?

### ISSUE THREAD

**Issue by laimonassutkus** [vercel/next.js#72176]
### Link to the code that reproduces this issue

https://github.com/laimonassutkus/VercelWaitUntilRemixExample

### To Reproduce

1. Create a new Remix.run app.
2. Create a route with **loader** and **action** methods.
3. Create a separate logging.tsx file that is not part of the routes.
4. Inside logging.tsx has a synchronous method info and an asynchronous method that actually logs to console.
5. Asynchronous method call is wrapped with the waitUntil method.
6. Using the logger in a route e.g. logger.info("Hello") does not work - waitUntil does not make the serverless function to not exit.

### Current vs. Expected behavior

- Console.log not getting called.
- Console.log is getting called.

### Provide environment information

```bash
- Remix.run
- Typescript
- Deploying on Vercel with waitUntil

- LOCALLY IT WORKS SUPER FINE WITHOUT BUGS !!!
```

### Which area(s) are affected? (Select all that apply)

Performance

### Which stage(s) are affected? (Select all that apply)

Vercel (Deployed)

### Additional context

```typescript
import { waitUntil } from "@vercel/functions"

async function asyncLog(log: string) {

    // We force the log to be printed only after some time.
    // Most importantly - AFTER the response is given.
    await new Promise(resolve => {
        setTimeout(resolve, 10_000)
    })

    console.log(log)
}

export default function info(log: string) {
    waitUntil(asyncLog(log))
}
```

**Comment by timneutkens** [comment]
This is the Next.js project repository. I'm guessing you're looking for Vercel support or the Vercel community: https://vercel.community/ 🙂
